File: listas-logpp/lista 2/lista 2 sala/questao 2.py
nota1 = float(input("Digite sua primeira nota:"))
nota2 = float(input("Digite sua segunda nota:"))
media = (nota1 + nota2) / 2
if media >= 6:
    print(f"Voce foi aprovado! sua nota foi de {media}")
else:
    exame = float(input("Digite sua nota de exame:"))
    media2 = (media + exame) / 2
    if media2 >= 5:
        print(f"Voce foi aprovado! sua nota foi de {media2}")
    else:
         print(f"Infelizmente voce nao foi aprovado sua media foi de {media2}")

File: listas de exercicio/listas 2/lista 2 sala C/questao 2.c
#include <stdio.h>

int main(){
    float N1, N2, M, NE, MN;
    printf("Digite sua primeira nota:");
    scanf("%f", &N1);
    printf("Digite sua segunda nota:");
    scanf("%f", &N2);
    M = (N1 + N2)/ 2;
    if (M >= 6)
    {
          printf("Parabens voce foi aprovado sua media foi de %.2f", M);
    } else{
        printf("Nao atingiu nota esperada, insira nota do exame:");
        scanf("%f", &NE);
        MN = (M + NE) / 2;
    }if (MN >=5){
        printf("Parabens voce foi aprovado sua media foi de %.2f", MN);
    } else{
         printf("Nao atingiu nota esperada");
    }
}